XMonad/Util/Hacks.hs

```diff
@@ -0,0 +1,67 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  XMonad.Util.Hacks
+-- Copyright   :  (c) 2020 Leon Kowarschick
+-- License     :  BSD3-style (see LICENSE)
+--
+-- Maintainer  :  Leon Kowarschick. <thereal.elkowar@gmail.com>
+-- Stability   :  unstable
+-- Portability :  unportable
+--
+-- This module is a collection of random fixes, workarounds and other functions
+-- that rely on somewhat hacky implementations which may have unwanted sideeffects.
+--
+-- Import this module as qualified like so:
+--
+-- > import qualified XMonad.Util.Hacks as Hacks
+--
+-- and then use the functions you want as described in their respective documentation.
+--
+-----------------------------------------------------------------------------
+
+module XMonad.Util.Hacks
+  ( -- * Windowed fullscreen
+    -- $windowedFullscreenFix
+    windowedFullscreenFixEventHook
+  ) where
+
+
+import XMonad
+import Data.Monoid (All(All))
+import Control.Monad (when)
+
+
+-- $windowedFullscreenFix
+-- Windowed fullscreen describes the behaviour in which XMonad,
+-- by default, does not automatically put windows that request being fullscreened
+-- into actual fullscreen, but keeps them constrained
+-- to their normal window dimensions, still rendering them in fullscreen.
+--
+-- With chromium based applications like Chrome, Discord and others this
+-- can cause issues, where the window does not correctly see the size of the window
+-- when displaying the fullscreen content, thus cutting off the window content.
+--
+-- This module works around that issue by forcing the window to recalculate their
+-- dimensions after initiating fullscreen, thus making chrome-based applications
+-- behave properly when in windowed fullscreen.
+--
+-- Usage:
+-- add to handleEventHook as follows:
+--
+-- > handleEventHook = handleEventHook def <+> windowedFullscreenFixEventHook
```

Comment:
May want to write `Hacks.windowedFullscreenFixEventHook` here now, in accordance with the top module documentation